iree/tools/run_mlir_main.cc

```diff
@@ -258,6 +265,21 @@ StatusOr<std::string> PrepareModule(
   return binary_contents;
 }
 
+// Returns a splitted input values from `filename` using comma as separater.
```

Comment:
If we're going to be reading from a file anyway, it seems like having these be newline-separated would be better. That would also allow commas in the input tensors themselves

Reply:
Yes, it's better. Let's do it. I was thinking to follow the format in iree-run-module.

btw, I'm not sure if it ("\n") is working on Windows... (we could worry about it later?)